cypress/e2e/map/map.cy.js

```diff
@@ -657,6 +680,9 @@ describe('Map interactions', () => {
         ).as('shapefilesApiRequest')
 
         cy.visit('/')
+        // After the react update, I needed to put this wait in or the test stalled and never moved on
+        // eslint-disable-next-line cypress/no-unnecessary-waiting
```

Comment:
This shows up in a few places, could it be moved to a global config?

Reply:
We don't like to globally disable things like this, that are recommended by the library we are using. Instead we disable it line by line when we don't have any other choice. In this instance, leaflet just needs a hair more time to process what we clicked, and we need to wait for that to happen. 

So this specific instance is a conscious choice, and we want any future `wait` calls to make that decision as well, instead of just accepting any `wait`s devs might add later